src/linear_algebra/basic.lean

```diff
@@ -2573,6 +2573,11 @@ def quot_equiv_of_eq (h : p = q) : p.quotient ≃ₗ[R] q.quotient :=
   ..@quotient.congr _ _ (quotient_rel p) (quotient_rel q) (equiv.refl _) $
     λ a b, by { subst h, refl } }
 
+@[simp]
+lemma quot_equiv_of_eq_mk (h : p = q) (x : M) :
+  submodule.quot_equiv_of_eq p q h (submodule.quotient.mk x) = submodule.quotient.mk x :=
+rfl
```

Comment:
Is there a similar lemma you can add for `ideal` and `(add_)subgroup`? It would probably make sense to add them all in the same PR.

Reply:
The one for `ideal` is ready to go, I'll quickly write the one for `(add_)subgroup` and add both to the PR